src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IForEachLoopStatement.cs

```diff
@@ -29,7 +29,9 @@ static void Main()
 }
 ";
             string expectedOperationTree = @"
-IForEachLoopStatement (Iteration variable: System.String value) (LoopKind.ForEach) (OperationKind.LoopStatement) (Syntax: 'foreach (st ... }')
+IForEachLoopStatement (LoopKind.ForEach) (OperationKind.LoopStatement) (Syntax: 'foreach (st ... }')
+  Locals: Local_1: System.String value
+  LoopControlVariable: ILocalReferenceExpression: value (OperationKind.LocalReferenceExpression, Type: System.String, Constant: null) (Syntax: 'foreach (st ... }')
```

Comment:
These `ILocalReferenceExpression` nodes for `LoopControlVariable` should probably have `IsDeclaration=true` which @333fred is going to add. I will file a separate issue to track this work.

Reply:
Filed https://github.com/dotnet/roslyn/issues/21828